.github/workflows/test.yml.jinja

```diff
@@ -0,0 +1,59 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+
+name: Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      os-variant:
+        default: '{{github_linux_image}}'
+        type: string
+      python-version:
+        default: '{{min_python}}'
+        type: string
+      tox-env:
+        default: 'test'
+        type: string
+      pip-recipe:
+        default: 'requirements/ci.txt'
+        type: string
+      coverage-report:
+        default: false
+        type: boolean
+  workflow_call:
+    inputs:
+      os-variant:
+        default: '{{github_linux_image}}'
+        type: string
+      python-version:
+        default: '{{min_python}}'
+        type: string
+      tox-env:
+        default: 'test'
+        type: string
+      pip-recipe:
+        default: 'requirements/ci.txt'
+        type: string
+      coverage-report:
+        default: false
+        type: boolean
+
+jobs:
+  test:
+    name: tests ${{ inputs.os-variant }}, py${{ inputs.python-version }}
+    runs-on: ${{ inputs.os-variant }}
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v3
+        with:
+          python-version: ${{ inputs.python-version }}
```

Comment:
@YooSunYoung This is broken, need to wrap in a `raw` section, otherwise jinja is not happy. Can you provide a fix?